src/tracers/no_tracer.rs

```diff
@@ -0,0 +1,34 @@
+use super::tracer::Tracer;
+use crate::state::VMState;
+use crate::{execution::Execution, Opcode};
+
+#[derive(Default)]
+pub struct NoTracer {}
+
+impl Tracer for NoTracer {
```

Comment:
It's ok to leave functions empty inside of this `Tracer`.

Reply:
This tracer is meant to be used when the client doesn't provide a custom tracer.